DEVELOPMENT.md

```diff
@@ -0,0 +1,143 @@
+	
+  
+## Download & Install Jellyfin 'unstable' version
+  
+1. goto: https://jellyfin.org/downloads  
+2. select your Operating system and click the 'Unstable' link (the Template plugin usually rely on latest development artifacts such as: MediaBrowser.Common 10.7 for this release so for it to work i suggest to prefer the unstable [latest night build] release)  
+3. then click the 'combined' link (this will download the server and the web client together)  
+4. once the zip has downloaded - extract it to a folder (lets say on windows: `%UserProfile%\Downloads\jellyfin_20210101.9-unstable`)  
+5. copy the content into a folder where you wish to run the unstable version server from (lets say on windows: `C:\Program Files\Jellyfin`)  
+6. so now you have a folder called: `C:\Program Files\Jellyfin\jellyfin_20210101.9` and in it you will find:  
+	- jellyfin-web
+	- Resources
+	- wwwroot
+	- api-ms-win-core-console-l1-1-0.dll
+	- ...  
+7. rename the folder: jellyfin_20210101.9 to: Server so you will have the path: `C:\Program Files\Jellyfin\Server`  
+8. from that folder run the Jellyfin executable file, for windows it is: `C:\Program Files\Jellyfin\Server\Jellyfin.exe`  
+9. after running the server check if it is up by going to: http://localhost:8096  
+10. follow the wizard setup steps to finish installing the Jellyfin server  
+11. login to the server using the recently created user (that you created in the wizard), sometimes it will require to restart the Jellyfin.exe app  
+12. once logged in, click the 3-line menu button from the top left corner (sometimes reffered to as 'Hamburger')  
+13. Click the 'Dashboard' submenu button  
+14. scroll down and click the 'Plugins' submenu button  
+15. make sure you see plugins tiles there that are ready to be installed (and can't see your 'Template' plugin yet)  
+16. make sure the plugins folder exist (in windows: `%UserProfile%\AppData\Local\jellyfin\plugins`)  
+  
+  
+## Download the 'Template' plugin solution
+  
+1. goto: https://github.com/jellyfin/jellyfin-plugin-template  
+2. from the top bar click the 'Code' button and press the 'Download ZIP' sub menu button  
+3. once the zip has downloaded - extract it to a folder (lets say on windows: `%UserProfile%\Downloads\pluginTemplate`)  
+4. in that folder there should be a folder called: jellyfin-plugin-template-master and in it you should find:  
+	- Jellyfin.Plugin.Template
+	- .editorconfig
+	- .gitignore
+	- build.yaml
+	- Jellyfin.Plugin.Template.sln
+	- LICENSE
+	- README.md
+	- ...  
+5. move the extracted folder (jellyfin-plugin-template-master) to the folder you want the solution to run from (for example: `C:\workspace\jellyfin-plugin-template-master`), we will refer to this folder as: `[TEMPLATE_PLUGIN_PROJECT_ROOT_FOLDER]`  
+  
+  
+## Open the 'Template' plugin solution in latest 'Microsoft Visual Studio'
+  
+>
+> [!NOTE] 
+> For this section you can either use: '**Microsoft Visual Studio**',
```

Comment:
Visual Studio Code is not the free community version of Visual Studio. They are 2 completely different programs.
[Visual Studio Community Edition](https://visualstudio.microsoft.com/vs/community/) is the free community version of Visual Studio.